otx/algorithms/common/adapters/mmcv/configs/backbones/lite_hrnet_18.py

```diff
@@ -0,0 +1,59 @@
+"""Backbone config of OCR-Lite-HRnet-18."""
```

Comment:
I think this model config might be in each task adapters, but this is about design choice.
Let's discuss about this later